Dockerfile

```diff
@@ -15,6 +15,5 @@ RUN chmod +x /bin/docker
 
 COPY . /usr/src/app
 
-ENV FILESYSTEM_DIR /code
-
+ENV CODECLIMATE_DOCKER 1
```

Comment:
If not being run via docker, we can't use the container-side paths of `/code` and `/tmp/cc`, we have to set them as whatever the host side is. I couldn't find any way to do that without:

- Having two new environment variables for the container side (with defaults) making 4 total
- Having a single environment variable to represent the single conditional (as I've done)

I may be missing something; if you have something in mind, please describe it a little more concretely.